internal/cmd/flink/command_compute_pool_update.go

```diff
@@ -20,18 +20,17 @@ func (c *command) newComputePoolUpdateCommand() *cobra.Command {
 		RunE:              c.computePoolUpdate,
 		Example: examples.BuildExampleString(
 			examples.Example{
-				Text: `Update Flink compute pool "my-compute-pool" to 2 CFUs.`,
-				Code: "confluent flink compute-pool update my-compute-pool --cfu 2",
+				Text: `Update name and CFU count of a Flink compute pool.`,
+				Code: "confluent flink compute-pool update my-compute-pool --name new_name --cfu 2",
 			},
 		),
 	}
 
 	cmd.Flags().Int32("cfu", 0, "Number of Confluent Flink Units (CFU).")
+	cmd.Flags().String("name", "", "Name of the compute pool.")
```

Comment:
Nit: `--name` normally comes first.
```suggestion
	cmd.Flags().String("name", "", "Name of the compute pool.")
	cmd.Flags().Int32("cfu", 0, "Number of Confluent Flink Units (CFU).")
```

Reply:
I wonder if it would help to use `uint32` for `--cfu` here.